hash/risc0/bench/src/main.rs

```diff
@@ -0,0 +1,75 @@
+mod benches;
+use benches::{
+    sha256::sha_bench,
+    keccak::keccak_bench,
+    blake2b::blake2b_bench,
+    blake3::blake3_bench,
+    poseidon2_bn128::poseidon2_bn128_bench,
+    poseidon2_babybear::poseidon2_babybear_bench,
+};
+use rand::Rng;
+
+
+fn generate_bytes(size: usize) -> Vec<u8> {
+    let mut rng = rand::thread_rng();
+    (0..size).map(|_| rng.gen()).collect()
+}
+
+
+
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+
+    // let mut flag = 0;
+
+    let hash_type = &args[1];
```

Comment:
Check if there are enough arguments, and print a usage help when there are no or not enough arguments given.